src/lang/en.json

```diff
@@ -538,7 +538,7 @@
       "officeAddresses": "Office Addresses",
       "currentDirectors": "Current Directors",
       "partners": "Partners",
-      "proprietors": "Proprietors",
+      "proprietor": "Proprietor",
```

Comment:
There is only 1 proprietor.

Compare with old Filings UI: https://dev.business.bcregistry.gov.bc.ca/FM1130015?noRedirect=true